### DISCUSSION TITLE
How do I change the colors on the Mason screen?

### DISCUSSION THREAD

**Discussion by CCurl** [mason-org/mason.nvim#1427] [Q&A] [answered]
How do I change the colors on the Mason screen?

Maybe it's just me, but the green text on the pink background is completely un-readable.

**Comment by CCurl** [comment, marked as answer]
LOL ... never mind ... I installed the tokyonight color scheme and it's all good now.

**Reply by williamboman** [reply]
Hehe yeah the default highlight for floating windows is not very pleasant. The default highlight group for floating windows is `:hi NormalFloat`, which by default links to `:hi Pmenu`.
